web/src/app/app.component.ts

```diff
@@ -111,4 +113,12 @@ export class AppComponent {
     public showDoorbell(): void {
         (<any>window).doorbell.show();
     }
+
+    ngOnInit(): void {
+        this.authService.getLoginEmitter().subscribe((result: boolean) => {
+            if (result) {
+                this.router.navigate(['/']);
```

Comment:
I don't think it is needed to redirect the user to the homepage on login. 

Also I think an event emitter might be too much, this file has access to the `isAuthenticated` variable from the `AuthService` https://github.com/DashboardHub/PipelineDashboard/pull/1218/files#diff-9fb615cefc2705778e04e2ef31bec93bR24

It is difficult for me to test, because the issue of not having the user details appear until they perform an action seems random

Reply:
What should happen after login ? Where should it go instead homepage ?

Reply:
Can the user not stay on the page they are currently on?

For example, if they are view a dashboard, then they decide to log in, they should stay on the same dashboard